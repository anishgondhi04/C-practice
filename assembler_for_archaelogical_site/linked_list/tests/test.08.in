2
list 0 new
list 1 new
list 0 build 6 alpha beta gamma delta beta gamma 
list 1 build 4 beta gamma delta alpha
list 0 overlap 1
list 0 splice 1
list 1 print
list 0 new
list 1 new
list 0 build 6 alpha beta gamma delta beta gamma 
list 1 build 4 beta gamma delta alpha
list 1 overlap 0
list 1 splice 0
list 0 print
end

