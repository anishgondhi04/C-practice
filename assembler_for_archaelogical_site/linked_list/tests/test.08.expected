List 0: new list
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 0: build
 [ alpha beta gamma delta beta gamma ] (6 items)
List 1: build
 [ beta gamma delta alpha ] (4 items)
List 0: Overlap between lists 0 and 1 is 2
List 0: List 1 spliced to the end of list 0
 [ alpha beta gamma delta beta gamma delta alpha ] (8 items)
List 1: print
 [ beta gamma ] (2 items)
List 0: new list
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 0: build
 [ alpha beta gamma delta beta gamma ] (6 items)
List 1: build
 [ beta gamma delta alpha ] (4 items)
List 1: Overlap between lists 1 and 0 is 1
List 1: List 0 spliced to the end of list 1
 [ beta gamma delta alpha beta gamma delta beta gamma ] (9 items)
List 0: print
 [ alpha ] (1 items)
